packages/pigeon/test/java_generator_test.dart

```diff
@@ -41,7 +41,7 @@ void main() {
       name: 'Foobar',
       members: <String>[
         'one',
-        'two',
+        'twoThreeFour',
```

Comment:
We should also include a test with something that follows the Dart style guide camel case exception for two-letter acronyms. E.g., `remoteDB`.